packages/dcos-integration-test/extra/test_mesos.py

```diff
@@ -181,6 +182,19 @@ def get_ps_output():
         assert len(get_ps_output().split()) <= 4, 'UCR app has more than 4 processes running in its pid namespace'
 
 
+def test_memory_profiling(dcos_api_session):
+    # Test that we can fetch raw memory profiles
+    master_ip = dcos_api_session.masters[0]
+    profile_id = dcos_api_session.get(
```

Comment:
Do we authorize this or the `GET` below? I didn't see any updates on IAM resource ids. If not, this sounds pretty bad since we could leak sensitive data in `strict` mode.

If we don't authorize, I'd suggest to either add authorization before enabling in Mesos, or only adding the config options in `disabled` security mode.

@jgehrcke